src/borg/archiver.py

```diff
@@ -4361,6 +4392,9 @@ def define_borg_mount(parser):
         subparser.add_argument('location', metavar='REPOSITORY', nargs='?', default='',
                                type=location_validator(archive=False),
                                help='repository to create')
+        subparser.add_argument('--other-repo', metavar='OTHER_REPOSITORY',
+                               type=location_validator(archive=False),
+                               help='reuse the key material from the other repository')
```

Comment:
guess we'll reuse this "other repository" mechanism for other purposes also, thus the name is rather generic.